integration-tests/src/tests/client/features/access_key_nonce_for_implicit_accounts.rs

```diff
@@ -764,7 +764,9 @@ fn test_chunk_forwarding_optimization() {
     // Note: For nightly, which includes SingleShardTracking, this check is disabled because
     // we're so efficient with part forwarding now that we don't seem to be forwarding more
     // than it is necessary.
-    if !cfg!(feature = "nightly") && !cfg!(feature = "statelessnet_protocol") {
+    // TODO - Since the stabilization of Stateless Validation which includes the
+    // SingleShardTracking this test doesn't make sense anymore. We should remove it.
+    if !ProtocolFeature::SingleShardTracking.enabled(PROTOCOL_VERSION) {
```

Comment:
@jancionear You touched this last, can you confirm this is the right thing to do please?

Reply:
I only know what the comment says:
```
// Note: For nightly, which includes SingleShardTracking, this check is disabled because
// we're so efficient with part forwarding now that we don't seem to be forwarding more
// than it is necessary.
```
I don't know if it should be removed, maybe just modified?

I disabled it for stateless validation based on the fact that it was disabled with `SingleShardTracking`